Engineering sections separated (Issue #13)

5anmu opened the pull request:
- Added some JavaScript animation, just for desktop view, if we feel it interferes with accessibility we can remove it, haha
- I liked Elias's original border design (#29), I added shadow to make the sections pop a little more, not sure yet if I like it better than just a simple tasteful top divider
- #13

5anmu committed:
Engineering sections separated (Issue #13)
- Added some JavaScript animation, just for desktop view, if we
feel it interferes with accessibility we can remove it
- I liked Elias's original border design, I added shadow to make
the sections pop a little more, not sure yet if I like it better
than just the simple top divider

komali2 commented:
The border separation is ok but feels a bit... idk, harsh. Visually it doesn't make sense from a lighting perspective where the source of light is if we want to consider each thing a "card." See google's material design for really well-implemented "paper" and "vertical" concepts for drop shadow. 

Some kind of animation would be cool and is bog-standard for these kinds of sites, but ours is not fleshed out enough to really include as-is. Check when it's fullscreen on a 4k screen, it's kinda nauseating. Also for more narrow-width screens it will go past the right-side viewport border (become partially obscured). Also it only seems to run on engineering page I think? I can't get it to trigger on any others. Also the animation re-runs if I scroll up/down, for each card. It should only ever run once per card. 

If we were going to have an animation like this I would include a fade-in, have the "final point" be where everything is positioned on the live app right now, and have it run much more quickly, as well as run only once. Also, it should run on all pages, or none.

komali2 reviewed (changes requested):
see comment

5anmu committed:
Box shadow higher contrast sharper edges, top border removed
- Animation on all pages, runs once, fade in, anchored screen-left,
if it doesn't fit atm, then this PR can just focus on section styles

komali2 commented:
ok, i like the paper effect you did here, i'm not sure about the animation. i think we need to stop a second and just actually design our site before we make little changes like this, let's chat more on slack about it. gonna leave this open so we can easily reference and grab the box shadow which i think i like
